src/asm/trampoline-aarch64.S

```diff
@@ -22,8 +22,14 @@
  * THE SOFTWARE.
  */
 
+#if !defined(__APPLE__)
 .globl mmk_trampoline
 mmk_trampoline:
+#else
+.align 4
+.globl _mmk_trampoline
```

Comment:
At least for the symbol name, could you use the macros in mangling.h to have it use the leading underscore on __APPLE__?

Would it work if we aligned the trampoline to 4 or 8 bytes for all platforms on aarch64? I'd rather have one thing that works well for everyone, and we allocate trampolines on one page anyway, so we have space to spare.